semantic_segmentation/scripts/augment.py

```diff
@@ -10,6 +10,7 @@ def get_transform(train=False):
         augments.append(A.RandomRotate90(p=0.5))
 
     augments.append(A.Resize(190, 275))
+    augments.append(A.Normalize(mean=(0, 0, 0), std=(1, 1, 1)))
```

Comment:
please don't!
there are only 2 popular values for those:
1. imagenet mean, std
2. (0.5, 0.5, 0.5) for both

Reply:
should i go for the second one then? cause i didn't want to scale the data in pytorch later on additionally and the new function for converting to tensors doesn't convert to floats

Reply:
whatever - you don't need another normalization
expect your normalized data between -2..+2 after this normalization
if you chose 2nd you can _denormalize_ by multiplying by 0.5 and then adding 0.5 to the image (result will be approx 0..1)